src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -18,14 +19,21 @@ public class StatsHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/stats", "GET")]
         public Task Stats()
         {
-            using (ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            using (ContextPool.AllocateOperationContext(out DocumentsOperationContext context))            
             using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             using (context.OpenReadTransaction())
             {
                 var indexes = Database.IndexStore.GetIndexes().ToList();
                 var transformersCount = Database.TransformerStore.GetTransformersCount();
 
                 var stats = new DatabaseStatistics();
+                using (Database.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext transactionOperationContext))
+                using (transactionOperationContext.OpenReadTransaction())
+                {
+                    var record = Database.ServerStore.Cluster.ReadDatabase(transactionOperationContext, Database.Name);
+                    stats.Identities = record.Identities;
```

Comment:
Maybe this should go to a separate endpoint? Don't need to overload the DB stats.